src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -32,6 +32,7 @@ internal class DependenciesProjectTreeProvider :
         IProjectTreeProvider,
         IDependenciesTreeServices
     {
+        private readonly IDependencyTreeTelemetryService _treeTelemetryService;
```

Comment:
>IDependencyTreeTelemetryService [](start = 25, length = 31)

Could you please follow same patter for imported components in this file? Add a readonly property below constructor ..  It is just easier to read if code follows same pattern in the file ..